rust/sealevel/client/src/main.rs

```diff
@@ -1231,21 +1245,33 @@ fn process_token_cmd(mut ctx: Context, cmd: TokenCmd) {
             println!("{:?}", tx_result);
         }
         TokenSubCmd::EnrollRemoteRouter(enroll) => {
-            let enroll_instruction = HtInstruction::EnrollRemoteRouter(RemoteRouterConfig {
-                domain: enroll.domain,
-                router: enroll.router.into(),
-            });
-            let (token_account, _token_bump) =
-                Pubkey::find_program_address(hyperlane_token_pda_seeds!(), &enroll.program_id);
+            let instruction = enroll_remote_routers_instruction(
+                enroll.program_id,
+                ctx.payer_pubkey,
+                vec![RemoteRouterConfig {
+                    domain: enroll.domain,
+                    router: enroll.router.into(),
+                }],
+            )
+            .expect("Should build instruction correctly");
+
+            ctx.new_txn().add(instruction).send_with_payer();
+        }
+        TokenSubCmd::SetDestinationGas(TokenSetDestinationGas {
```

Comment:
curious why this was necessary to add? `warp-route deploy` should do this for you automatically instead of needing to call this manually?